ui/webui/resources/polymer_overriding.ts

```diff
@@ -366,3 +367,6 @@ export function html(strings, ...values) {
   htmlStrings.raw = htmlStrings
   return polymerHtml(htmlStrings).content.cloneNode(true)
 }
+
+// Overrides for all pages
+RegisterStyleOverrideName('cr-toast', 'cr-toast-override')
```

Comment:
This seems to be the only place `RegisterStyleOverrideName` is used, so I wouldn't be surprised if it doesn't work....

could you convert to using

```
RegisterStyleOverride('cr-toast', html`<style>...</style>`)
```?

Reply:
Alternatively, you might need to include the `cr_toast_override` in `cr_toast` via a patch.

Reply:
Unfortunately, `RegisterStyleOverride` also doesn't work.
```
RegisterStyleOverride(
  'cr-toast',
  html`
    <style>
      :host {
        box-shadow: 0px 4px 8px -1px #0000001A !important;
      }
    </style>
  `
)
```
and adding this to `cr_toast.css` doesn't work
` #import=//resources/brave/overrides/cr_toast_override.css.js`
Maybe I'm missing something or `cr_taost` is lit element?

Reply:
I'm thinking to update only bg/fg colors of toast w/o changing shadow. it's beyond my knowledge.

Reply:
Oh... `cr-toast` is indeed a `LitElement`. That's going to make things a bit more complicated. I've just landed a commit in `cr126` which let's us override styles in Lit.

https://github.com/brave/brave-core/pull/23525

So yeah, unfortunately I think we won't be able to override until `cr126` is in.